proto/rig/api/v1/capsule/service.proto

```diff
@@ -77,6 +77,41 @@ service Service {
 
   rpc GetStatus(GetStatusRequest) returns (GetStatusResponse) {}
   rpc GetRevision(GetRevisionRequest) returns (GetRevisionResponse) {}
+  rpc WatchStatus(WatchStatusRequest) returns (stream WatchStatusResponse) {}
+}
+
+message WatchStatusRequest {
+  // The capsule to watch the status of.
+  string capsule_id = 1;
+  // The project in which the capsule lives.
+  string project_id = 2;
+  // The environment to watch the status of.
+  string environment_id = 3;
+}
+
+message WatchStatusResponse {
+  message Interfaces {
+    repeated api.v1.capsule.InterfaceStatus interfaces = 1;
+  }
+
+  message ConfigFiles {
+    repeated api.v1.capsule.ConfigFileStatus config_files = 1;
+  }
+
+  message CronJobs {
+    repeated api.v1.capsule.CronJobStatus CronJobs = 1;
+  }
+
+  oneof reponse {
```

Comment:
It's probably easier to just have it return everything each time? Otherwise it will be quite annoying to maintain :)